pyproject.toml

```diff
@@ -89,4 +94,4 @@ ignore_missing_imports = true
 warn_unused_configs = true
 
 [tool.black]
-line-length = 110
+line-length = 110
```

Comment:
Make sure you do not commit changes you are not supposed to commit (there is no need for this).